packages/upset/src/components/ElementView/ElementVisualization.tsx

```diff
@@ -34,14 +35,17 @@ export const ElementVisualization = () => {
   const { actions }: {actions: UpsetActions} = useContext(ProvenanceContext);
 
   /**
-   * Hooks
+   * Internal State
    */
 
   const draftSelection = useRef(elementSelection);
   const vegaSpec = useMemo(
     () => generateVega(scatterplots, histograms, selectColor, elementSelection),
     [scatterplots, histograms, selectColor, elementSelection],
   );
+  const data = useMemo(() => ({
+    elements: Object.values(JSON.parse(JSON.stringify(items))),
```

Comment:
Would a `structuredClone` here be better than the parse/stringify hack?

Reply:
+1